README.md

```diff
@@ -730,76 +730,130 @@ classDiagram
     CSSBlockList <|-- Document: inheritance
+    Document ..> OutputFormat: dependency
+    Document ..> ParserState: dependency
     AtRule <|.. Import: realization
+    Import ..> OutputFormat: dependency
     Positionable <|.. Import: realization
+    Import ..> URL: dependency
     AtRule <|.. KeyFrame: realization
     CSSList <|-- KeyFrame: inheritance
+    KeyFrame ..> OutputFormat: dependency
     Selector <|-- KeyframeSelector: inheritance
+    LineName ..> OutputFormat: dependency
+    LineName ..> ParserState: dependency
     ValueList <|-- LineName: inheritance
     SourceException <|-- OutputException: inheritance
     OutputFormat ..> OutputFormatter: dependency
+    OutputFormatter ..> Commentable: dependency
+    OutputFormatter ..> InvalidArgumentException: dependency
     OutputFormatter ..> OutputFormat: dependency
+    Parser ..> Document: dependency
     Parser ..> ParserState: dependency
+    ParserState ..> Anchor: dependency
+    ParserState ..> Comment: dependency
     ParserState ..> Settings: dependency
+    ParserState ..> SourceException: dependency
+    ParserState ..> UnexpectedEOFException: dependency
+    ParserState ..> UnexpectedTokenException: dependency
     Value <|-- PrimitiveValue: inheritance
+    Renderable ..> OutputFormat: dependency
     CSSElement <|.. Rule: realization
     Commentable <|.. Rule: realization
+    Rule ..> OutputFormat: dependency
+    Rule ..> ParserState: dependency
     Positionable <|.. Rule: realization
     Rule ..> RuleValueList: dependency
+    RuleContainer ..> Rule: dependency
     CSSElement <|.. RuleSet: realization
     CSSListItem <|.. RuleSet: realization
+    RuleSet ..> OutputFormat: dependency
+    RuleSet ..> ParserState: dependency
     Positionable <|.. RuleSet: realization
     RuleSet ..> Rule: dependency
     RuleContainer <|.. RuleSet: realization
+    RuleSet ..> UnexpectedValueException: dependency
     ValueList <|-- RuleValueList: inheritance
+    Selector ..> OutputFormat: dependency
     Renderable <|.. Selector: realization
+    Size ..> OutputFormat: dependency
+    Size ..> ParserState: dependency
     PrimitiveValue <|-- Size: inheritance
     Exception <|-- SourceException: inheritance
     Positionable <|.. SourceException: realization
     URL ..> CSSString: dependency
+    URL ..> OutputFormat: dependency
+    URL ..> ParserState: dependency
     PrimitiveValue <|-- URL: inheritance
     UnexpectedTokenException <|-- UnexpectedEOFException: inheritance
     SourceException <|-- UnexpectedTokenException: inheritance
     CSSElement <|.. Value: realization
+    Value ..> CSSFunction: dependency
+    Value ..> ParserState: dependency
     Positionable <|.. Value: realization
+    Value ..> RuleValueList: dependency
+    Value ..> UnexpectedTokenException: dependency
+    ValueList ..> OutputFormat: dependency
     Value <|-- ValueList: inheritance
-
-    CSSList ..> CSSList: dependency
-    CSSList ..> Comment: dependency
```

Comment:
Looks like it's losing dependencies that are class properties via traits.  The `CommentContainer` trait was added for classes that have `Comment`s.  Given that traits are not included in the class diagram, this looks to me like a bug in the generator (for which I think the better fix would be to include traits in their own right, like interfaces and classes).